plugins/web-terminal/src/components/TerminalComponent/TerminalComponent.tsx

```diff
@@ -31,22 +37,32 @@ const useTerminalStyles = makeStyles({
 
 interface IFormData {
   token: string | undefined;
+  namespace: string | undefined;
 }
 
 export const TerminalComponent = () => {
+  const config = useApi(configApiRef);
+
   const [websocketRunning, setWebsocketRunning] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
+  const [isHidden, setIsHidden] = React.useState(true);
+  const defaultNamespace =
+    config.getOptionalString('webTerminal.defaultNamespace') ??
+    'openshift-terminal';
```

Comment:
Move the `openshift-terminal` into a separate const. This is an important setting/value that should not be inlined and hidden somewhere within a component.

Also `config.getOptionalString('webTerminal.defaultNamespace') ?? 'openshift-terminal';` should be a helper function exposed somewhere from utils/elsewhere. You don't want to potentially repeat this code in different ways. This should behave consistently.